src/ontology/mondo-edit.obo

```diff
@@ -334863,15 +334863,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 
 [Term]
 id: MONDO:0014598
-name: developmental and epileptic encephalopathy, 31
-def: "Any early infantile epileptic encephalopathy in which the cause of the disease is a mutation in the DNM1 gene." [MONDO:patterns/disease_series_by_gene]
+name: developmental and epileptic encephalopathy, 31A
+def: "Any developmental and epileptic encephalopathy in which the cause of the disease is a heterozygous mutation in the DNM1 gene." [https://orcid.org/0000-0002-0587-4693, MONDO:patterns/disease_series_by_gene, OMIM:616346]
 subset: gard_rare {source="GARD:16094", source="MONDO:GARD"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "DEE31" BROAD ABBREVIATION [OMIM:616346]
+synonym: "DEE31A" EXACT [OMIM:616346]
 synonym: "developmental and epileptic encephalopathy 31" BROAD [OMIM:616346, OMIM:genemap2]
+synonym: "Developmental and epileptic encephalopathy 31A, autosomal dominant" EXACT [https://orcid.org/0000-0001-9310-0163]
```

Comment:
minor thing - labels should be lowercase unless they are a proper name or an abbreviation

Reply:
Comment addressed